spec/mainadditions.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <meta charset="utf8">
 
-<emu-clause id="sec-temporal-legacy-date-objects">
+<emu-clause id="sec-temporal-ecma262-amendments">
   <h1>Amendments to the ECMAScript® 2023 Language Specification</h1>
```

Comment:
Should this get a reference to the exact commit used now that it has more complicated changes?